services/kafka.ts

```diff
@@ -1,4 +1,3 @@
-// import app from '@adonisjs/core/services/app'
 import app from '@adonisjs/core/services/app'
 import { Kafka } from '../src/index.ts'
```

Comment:
Feels like this should be an:
```suggestion
import type { Kafka } from '../src/index.ts'
```

But weird things happen if you do that?